apps/verticalhome/js/app.js

```diff
@@ -158,6 +193,179 @@
 
     },
 
+    saveAllPinnedApps: function(){
+
+      var pinnedItems= app.pinnedAppsManager.items;
+      if(this.deletedApps.length>0){
+        for(var x = 0; x < pinnedItems.length; x++){
+          if(pinnedItems[x].element.classList.contains('hidden-item')){
```

Comment:
It would be more convenient if you add comments for functions. It makes easier to understand what function is for.